app/components/layout/header/index.tsx

```diff
@@ -23,6 +25,12 @@ export default function Header({
   hidePageDescription?: boolean;
   hideBreadcrumbs?: boolean;
   classNames?: string;
+  asset?: {
```

Comment:
Okey, so this approach is okey but it feels a bit hardcoded. What I would rather do is adjust the header component that the title is either a string or a react node. So we can just pass some react code. Kinda similar like it works with children in react.